packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -202,6 +201,9 @@ export const getPublicEvent = async (
                 }
               : {
                   username,
+                  profiles: {
```

Comment:
Organization username was servable from app.cal.com. It's not a problem as such but it's still an unexpected bug.